python/ray/tests/test_actor_advanced.py

```diff
@@ -1294,6 +1294,32 @@ def hi(self):
         ray.get_actor("actor_name")  # => errors
 
 
+def test_get_actor_after_killed(shutdown_only):
+    ray.init(num_cpus=2)
+
+    @ray.remote
+    class A:
+        def ready(self):
+            return True
+
+    actor = A.options(name="namespace/actor", lifetime="detached").remote()
+    ray.kill(actor)
+    with pytest.raises(ValueError):
+        ray.get_actor("namespace/actor")
+
+    actor = A.options(
+        name="namespace/actor_2", lifetime="detached",
+        max_restarts=1).remote()
+    ray.kill(actor, no_restart=False)
+    assert ray.get(ray.get_actor("namespace/actor_2").ready.remote())
+
+    # TODO(sang): This currently doesn't pass without time.sleep.
+    # ray.kill(actor, no_restart=False)
```

Comment:
Not sure why it doesn't pass, but this is a lower priority bug to fix (since it doesn't crash gcs)

Reply:
it might be fixed after Yi's PR. I will revisit it later.

Reply:
Should we have a new ticket for this? Or it's tracked by the existing one.

Reply:
Yeah I think so. I think it is a lower priority as it is not hanging + crashing, but let me file an issue after the PR is merged.

Reply:
Maybe you can try with the latest master and if it's fixed we can just let it go. Otherwise, we probably need a new ticket.

Reply:
not fixed unfortunately..